ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/Decision/DeclineReason.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@page "/task-list/{id:int}/decision/declined-reason"
+@using ApplyToBecome.Data.Models.AdvisoryBoardDecision
+@model ApplyToBecomeInternal.Pages.TaskList.Decision.DeclineReasonModel
+@{
+	ViewData["Title"] = "Why was this project declined?";
+}
+
+@section BeforeMain
+{
+	<partial name="_BackLink" model="@Model.BackLinkModel"/>
+	<partial name="_ErrorSummary"/>
+}
+
+<span id="selection-span" class="govuk-caption-l">@Model.SchoolName</span>
+
+<form method="post">
+
+	<div class="govuk-form-group">
+		<fieldset class="govuk-fieldset" aria-describedby="declined-reasons-hint">
+			<legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+				<h1 class="govuk-fieldset__heading">
+					Why was this project declined?
+				</h1>
+			</legend>
+
+			<div id="declined-reasons-hint" class="govuk-hint">
+				Select at least one option.
+			</div>
+
+			<div class="govuk-checkboxes" data-module="govuk-checkboxes">
+				@foreach (var declineOption in Model.DeclineOptions)
+				{
+					<div class="govuk-checkboxes__item">
+						<input name="DeclinedReasons" 
+						       checked="@Model.UI.IsChecked(declineOption)" 
+						       class="govuk-checkboxes__input" 
+						       id="@Model.UI.IdFor("declined-reasons", declineOption)" 
+						       type="checkbox" 
+						       value="@Model.UI.ValueFor(declineOption)" 
+						       aria-controls="@Model.UI.IdFor("conditional-declined-reasons", declineOption)" 
+						       aria-expanded="false">
+						<label class="govuk-label govuk-checkboxes__label" 
+						       for="@Model.UI.IdFor("declined-reasons", declineOption)">
+							@declineOption.ToDescription()
+						</label>
+					</div>
+
+					<div class="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" id="@Model.UI.IdFor("conditional-declined-reasons", declineOption)">
```

Comment:
Oh nice, this is handy. Didn't know you could do that.

Reply:
I meant the hidden classes. Presumably there's some js in the gov package that handles setting it to visible.